shardingsphere-sql-parser/shardingsphere-sql-parser-statement/src/test/java/org/apache/shardingsphere/sql/parser/sql/dialect/handler/ddl/CreateTableStatementHandlerTest.java

```diff
@@ -42,4 +50,31 @@ public void assertIfNotExists() {
         assertFalse(CreateTableStatementHandler.ifNotExists(new SQLServerCreateTableStatement()));
         assertFalse(CreateTableStatementHandler.ifNotExists(new SQL92CreateTableStatement()));
     }
+
+    @Test
+    public void assertSelectStatement() {
```

Comment:
assertGetSelectStatement may be better.

Reply:
Makes sense, done - https://github.com/apache/shardingsphere/pull/20788/commits/b879baf960de092e8bcb6f7c27ad69f09b89c300